.npmrc

```diff
@@ -0,0 +1,2 @@
+# using yarn
+package-lock = false
```

Comment:
@priley86 Is `yarn.lock = true` implicit here?

Reply:
@AparnaKarve i'm not sure - maybe you have another suggestion? This was the setting in ui-classic.

Reply:
Maybe it is implicit.
Let's go with this setting, since classic-ui also has it.

Reply:
`yarn.lock` is generated always when you run yarn, and there's no real way to disable it,
but we can just keep `.gitignore`ing it, same as ui-classic.